cadet/cadet.py

```diff
@@ -192,7 +185,11 @@ def cadet_path(cls, cadet_path: os.PathLike) -> None:
         """
         cadet_path = Path(cadet_path)
 
-        warnings.warn("Deprecation warning: Support for setting cadet.cadet_path will be removed in a future version.")
+        warnings.warn(
+            "Support for setting cadet.cadet_path will be removed in a future version. "
+            "TODO: What alternative should be used?",
```

Comment:
[TODO]: @ronald-jaepel I forgot, but what alternative should be used in the future?